lang/java/reef-io/src/main/java/org/apache/reef/io/network/impl/NetworkConnectionFactory.java

```diff
@@ -77,9 +77,23 @@
   }
 
   Link<NetworkConnectionServiceMessage<T>> openLink(final Identifier remoteId) throws NetworkException {
+    // TODO : Remove below if statement after the deprecated methods in 0.13 are removed.
```

Comment:
From checkstyle point of view, please include reference to corresponding JIRA in TODO: `TODO[JIRA REEF-637]`. This will reduce the amount of work for the person triaging TODOs as part of https://issues.apache.org/jira/browse/REEF-519.